### PR TITLE
WX-1411 Require preinstalled `jq` and `curl` for just-in-time SAS feature

### PR DESCRIPTION
**Pull request by aednichols**

### CODE REVIEW

File: supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala [hunk 1]
```diff
@@ -86,32 +86,16 @@ object TesAsyncBackendJobExecutionActor {
...
-       |    apt-get -y update && apt-get -y install curl
-       |    if [ $$? -ne 0 ]; then
-       |      echo "Error: Failed to install curl via apt-get."
-       |      exit 1
-       |    fi
-       |  else
-       |    echo "Error: apt-get is not available, and curl is not installed."
-       |    exit 1
-       |  fi
+       |  echo "Error: user image must include `jq` for just-in-time SAS token generation, but it is not installed."
```
Comment (jgainerdewar) [inline supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala:92]: I think these two error messages need to be reversed.

Comment (aednichols) [reply supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala:92]: 🤦‍♂️ thank you!

**Review by jgainerdewar — APPROVED** [review]

**Comment by THWiseman** [comment]
LGTM! I updated [this doc](https://docs.google.com/document/d/1X1mpcUtsukeWez82UpXAXkX7SJpcSB-pZk1lFwd6LUA/edit) to reflect the change here.

**Review by THWiseman — APPROVED** [review]

**Comment by aednichols** [comment]
> I updated this doc

Thanks!

**Pull request merged**